app/root/root.css

```diff
@@ -984,6 +984,11 @@ code .comment {
   background: #cfd8dc;
 }
 
+.role-badge.HOSTED_BAZEL {
+  background: #a1a6a8;
```

Comment:
Maybe we can use a more exciting color here like green 500 from https://materialui.co/colors/

Reply:
What do you think of this? The green seemed a little too close to a "success" color
![Screenshot 2024-06-11 at 3 41 01 PM](https://github.com/buildbuddy-io/buildbuddy/assets/13951661/8929dae6-6007-4850-abf9-eec38da8c529)